packages/Checkboxes/__tests__/Checkbox.test.tsx

```diff
@@ -40,67 +42,53 @@ const defaultProps = (override = {}) => ({
   ...override,
 });
 
-describe("Checkboxes component", () => {
-  test("should be defined and renders correctly (snapshot)", () => {
-    const wrapper = shallow(<Checkboxes {...defaultProps()} />);
-
-    expect(wrapper).toBeDefined();
-    expect(wrapper).toMatchSnapshot();
+describe('Checkboxes component', () => {
+  it('should be defined and renders correctly (snapshot)', () => {
+    const { asFragment } = render(<Checkboxes {...defaultProps()} />);
+    expect(asFragment()).toMatchSnapshot();
   });
 
-  test("should render and toggle single checkbox", () => {
+  it('should render and toggle single checkbox', () => {
     const override = {
       options: single,
       returnBoolean: true,
     };
 
-    const wrapper = mount(<Checkboxes {...defaultProps(override)} />);
+    render(<Checkboxes {...defaultProps(override)} />);
 
-    expect(wrapper.find("input").length).toBe(1);
+    const checkboxArray = screen.getAllByRole('checkbox');
 
-    expect(wrapper.find("input").at(0).prop("checked")).toBe(false);
-
-    wrapper.find("input").at(0).simulate("click");
+    expect(checkboxArray.length).toBe(1);
+  });
 
-    expect(wrapper.find("input").at(0).prop("checked")).toBe(true);
+  it('should render a disabled checkbox', () => {
+    const { asFragment } = render(<Checkboxes {...defaultProps()} />);
+    expect(asFragment()).toMatchSnapshot();
   });
 
-  test("should render multiple checkboxes and toggle", () => {
-    let selectedBoxLabel: string = "";
+  it('should render multiple checkboxes and toggle', () => {
+    let selectedBoxLabel = '';
 
     type TValue = object | any;
 
     const onChange = ({ value: [{ label }] }: TValue) => {
       selectedBoxLabel = label;
     };
 
-    let override = {
+    const override = {
       onChange,
     };
 
-    const { rerender, getByTestId } = render(
-      <Checkboxes {...defaultProps(override)} />
-    );
+    render(<Checkboxes {...defaultProps(override)} />);
+    fireEvent.click(screen.getByTestId('1-checkbox'));
 
-    fireEvent.click(getByTestId("checkbox-1"));
-
-    expect(selectedBoxLabel).toEqual("first");
-
-    override = {
-      ...override,
-      ...{
-        options: single,
-      },
-    };
-
-    rerender(<Checkboxes {...defaultProps(override)} />);
+    expect(selectedBoxLabel).toEqual('first');
   });
 
-  test("can't interact when checkbox is disabled", () => {
-    const wrapper = mount(<Checkboxes {...defaultProps()} />);
-
-    wrapper.find(".form-field").at(2).simulate("click");
-
-    expect(wrapper.find("input").at(2).prop("checked")).toBe(false);
+  it("can't interact when checkbox is disabled", () => {
+    render(<Checkboxes {...defaultProps()} />);
+    const checkbox = screen.getByTestId('1-checkbox');
```

Comment:
Shouldn't this test be checking `3-checkbox` as `1-checkbox` isn't disabled? 

I think this is working because there the Checkboxes component must be changing the options object directly in the previous test so `1-checkbox` is already checked when rendered in this test. 

We should potentially add this as a bug story? We could also fix this test by storing `options` in  `defaultProps`

Reply:
`fixme`

Reply:
https://byte-9.atlassian.net/browse/BZ2-2453